### PR TITLE
Add a "default" button to the loadsave window

### PR DESCRIPTION
**Pull request by SijmenSchoon**
This button will jump directly to the default save directory.

I haven't tested this with tracks, because I have no idea how to open a loadsave window for tracks.

### CODE REVIEW

File: src/windows/loadsave.c [hunk 1]
```diff
@@ -45,19 +45,21 @@ enum {
...
+	{ WWT_CAPTION,		0,		1,					WW - 2,			1,			14,			STR_NONE,			  STR_WINDOW_TITLE_TIP },
+	{ WWT_CLOSEBOX,		0,		WW - 13,			WW - 3,			2,			13,			STR_CLOSE_X,		  STR_CLOSE_WINDOW_TIP }, //Window close button
+	{ WWT_CLOSEBOX,		0,		105,				205,			36,			47,			2718,				  STR_NONE},	// Up
+	{ WWT_CLOSEBOX,		0,		206,				307,			36,			47,			2719,				  STR_NONE},	// New
+	{ WWT_CLOSEBOX,		0,		4,					(WW - 5) / 2,	50,			61,			STR_NONE,			  STR_NONE },	// Name
+	{ WWT_CLOSEBOX,		0,		(WW - 5) / 2 + 1,	WW - 5 - 1,		50,			61,			STR_NONE,			  STR_NONE },	// Date
+	{ WWT_SCROLL,		0,		4,					WW - 5,			61,			WH - 40,	2,					  STR_NONE },	// File list
+	{ WWT_CLOSEBOX,		0,		4,					200,			WH - 36,	WH - 18,	2707,				  STR_NONE }, // Use native browser
+	{ WWT_CLOSEBOX,     0,      4,                  104,            36,			47,			STR_LOADSAVE_DEFAULT, STR_LOADSAVE_DEFAULT_TIP }, // Go to default directory
 	{ WIDGETS_END }
```
Comment (SijmenSchoon) [inline src/windows/loadsave.c]: I have no idea what my editor did here...

**Comment by SijmenSchoon** [comment]
I'm just cleaning up the tab/space mess my editor created, hang on

File: data/language/en-GB.txt [hunk 2]
```diff
@@ -4199,6 +4199,8 @@ STR_5887    :Prefix
 STR_5888    :Suffix
 STR_5889    :Custom currency symbol
 STR_5890    :Enter the currency symbol to display
+STR_5898    :Default
```
Comment (marijnvdwerf) [inline data/language/en-GB.txt]: Why are you skipping 5891-5897?

**Comment by IntelOrca** [comment]
@SijmenSchoon go in to the track manager and click install new track. To save a track, click on a tracked ride, then go to measurements tab then click save.

File: data/language/en-GB.txt (hunk 2 above)
Comment (AaronVanGeffen) [inline data/language/en-GB.txt]: I presume it has to do with #3964, but that PR has since added a few more strings.

Comment (wolfreak99) [inline data/language/en-GB.txt]: just for the record i have stopped adding strings at 5899 and will likely not add any more. also, don't feel you need to skip any of the strings due to my pull requests, as if yours gets merged before mine, i'll merely update the string ids in my PR ^_^. 

I have a feeling that the devs don't want us to skip strings for other pull requests anyway (at least, this is what i am merely assuming, i'm not entirely sure on this), as the other pull requests aren't always guaranteed to be merged.

**Comment by Nubbie** [comment]
Congratz on getting the ID #4000 btw ;p

**Comment by SijmenSchoon** [comment]
I'll change the str ids and test track load/saving tomorrow

**Comment by SijmenSchoon** [comment]
@devs can be tested and merged now!

**Pull request merged**